### ISSUE TITLE
MinIO console unable to reach S3 interface

### ISSUE THREAD

**Issue by jadedeane** [truenas/charts#2577]
TrueNAS-SCALE-24.04.1.1, MinIO app 2023-03-13 chart 2.6.0 image minio/minio:RELEASE.2023-07-21T21-12-44Z.

When logging in to console:

```text
Post "https://nas1.domain:9000/": dial tcp 192.168.0.200:9000: i/o timeout
```

From within the MinIO pod:

```console
root@nas1[~]# k3s kubectl -n ix-minio exec -it minio-7bc8b845f4-bplbj -c minio -- bash
bash-4.4$ curl https://localhost:9000
curl: (51) SSL: no alternative certificate subject name matches target host name 'localhost'
bash-4.4$ curl https://192.168.0.200:9000
curl: (7) Failed to connect to 192.168.0.200 port 9000: Connection timed out
```

MinIO pod logs:

```console
root@nas1[~]# k3s kubectl -n ix-minio logs -f  minio-7bc8b845f4-bplbj -c minio
MinIO Object Storage Server
Copyright: 2015-2023 MinIO, Inc.
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Version: RELEASE.2023-07-21T21-12-44Z (go1.19.11 linux/amd64)

Status:         1 Online, 0 Offline.
S3-API: https://nas1.domain:9000
Console: https://nas1.domain:9002

Documentation: https://min.io/docs/minio/linux/index.html
Warning: The standard parity is set to 0. This can lead to data loss.

 You are running an older version of MinIO released 10 months ago
 Update: Run `mc admin update`
```

Using curl above to test socket connections (in lieu of natcat), localhost works (cert fail, expected), but IP (and by extension FQDN as well if used) fails.

This was working a few days ago, and problems seem to coincide with an automatic cert update (cert is valid):
```console
❯ echo | openssl s_client -showcerts -servername nas1.domain -connect nas1.domain:9000 2>/dev/null | openssl x509 -inform pem -noout -text
...
        Validity
            Not Before: Jun  3 18:41:47 2024 GMT
            Not After : Sep  1 18:41:46 2024 GMT
        Subject: CN=nas1.domain
```

tl;dr, MinIO pod console can't reach s3 interface within pod.

**Comment by jadedeane** [comment]
> Hello, from within the container you can't reach the host IP unless you use host network. Your client should be able to reach your host IP, and also be able to resolve the FQDN. Which I assume it should do already considering that it was working before.
> 
> ~Please try "editing" the minio app and saving without any changes.~ I see tha t the cert is picked up correctly, so this shouldn't bring any new results.
> 
> Can you share what curl shows from your client to minio?

Client curl? Sure, cert's valid, so all I'd expect would be complaining about JS:
```console
❯ curl https://nas1.domain:9002
<!doctype html><html lang="en"><head><meta charset="utf-8"/><base href="/"/><meta content="width=device-width,initial-scale=1" name="viewport"/><meta content="#081C42" media="(prefers-color-scheme: light)" name="theme-color"/><meta content="#081C42" media="(prefers-color-scheme: dark)" name="theme-color"/><meta content="MinIO Console" name="description"/><meta name="minio-license" content="agpl" /><link href="./styles/root-styles.css" rel="stylesheet"/><link href="./apple-icon-180x180.png" rel="apple-touch-icon" sizes="180x180"/><link href="./favicon-32x32.png" rel="icon" sizes="32x32" type="image/png"/><link href="./favicon-96x96.png" rel="icon" sizes="96x96" type="image/png"/><link href="./favicon-16x16.png" rel="icon" sizes="16x16" type="image/png"/><link href="./manifest.json" rel="manifest"/><link color="#3a4e54" href="./safari-pinned-tab.svg" rel="mask-icon"/><title>MinIO Console</title><script defer="defer" src="./static/js/main.90ab1516.js"></script><link href="./static/css/main.7f39beab.css" rel="stylesheet"></head><body><noscript>You need to enable JavaScript to run this app.</noscript><div id="root"><div id="preload"><img src="./images/background.svg"/> <img src="./images/background-wave-orig2.svg"/></div><div id="loader-block"><img src="./Loader.svg"/></div></div></body></html>
```

In browser, it was a ~3min spinning circle but finally loaded page. When attempting to login **from browser**, the following error was presented:

```text
Post "https://nas1.domain:9000/": dial tcp 192.168.0.200:9000: i/o timeout
```

![Screenshot 2024-06-17 at 9 51 16 AM](https://github.com/truenas/charts/assets/11843794/f5775d6e-85cb-4bf4-a25a-1d125d563345)

**Comment by jadedeane** [comment]
> Hmm I suppose you have set the domain field right?

"Set the domain field" where exactly? Within TrueNAS, yup, box steady state, no changes. All "native" (iSCSI, NFS< SMB) services working as expected. MinIO, which is my only chart application, is the outlier.

**Comment by stavros-k** [comment]
> > Hmm I suppose you have set the domain field right?
> 
> 
> 
> "Set the domain field" where exactly? Within TrueNAS, yup, box steady state, no changes. All "native" (iSCSI, NFS< SMB) services working as expected. MinIO, which is my only chart application, is the outlier.
> 
> 
> 
> The screenshot above reads to me as a simple socket failure.
> 
> 
> 
> As I mentioned previously, not remotely sure where this would die in, but things started failing around the time of automatic cert renewal.

Sorry, meant the Domain field in the MinIO Wizard
(Apps -> Select installed MinIO -> Edit)
This sets which URL will the browser use for redirections and such.

If this is not set, I'm more puzzled as to why it was working before

**Comment by jadedeane** [comment]
> > > Hmm I suppose you have set the domain field right?
> > 
> > 
> > "Set the domain field" where exactly? Within TrueNAS, yup, box steady state, no changes. All "native" (iSCSI, NFS< SMB) services working as expected. MinIO, which is my only chart application, is the outlier.
> > The screenshot above reads to me as a simple socket failure.
> > As I mentioned previously, not remotely sure where this would die in, but things started failing around the time of automatic cert renewal.
> 
> Sorry, meant the Domain field in the MinIO Wizard (Apps -> Select installed MinIO -> Edit) This sets which URL will the browser use for redirections and such.
> 
> If this is not set, I'm more puzzled as to why it was working before

Ah, yes, Apps -> MinIO -> Edit, FQDN (same as nas itself) is configured.

I only happened to notice Velero backups were failing, tried to log into MinIO, and well, here we are :)

**Comment by jadedeane** [comment]
I deleted and attempted a fresh MinIO install, and seeing this:
```console
root@nas1[~]# k3s kubectl -n ix-minio logs -f minio-5b54d975cd-xv7ln
Defaulted container "minio" out of: minio, minio-init-01-permissions (init)

API: SYSTEM()
Time: 23:10:27 UTC 06/17/2024
Error: unable to rename (/export/.minio.sys/tmp -> /export/.minio.sys/tmp-old/47383920-43da-4892-9252-5b8bc706937f) file access denied, drive may be faulty please investigate (*fmt.wrapError)
       6: internal/logger/logger.go:258:logger.LogIf()
       5: cmd/prepare-storage.go:88:cmd.bgFormatErasureCleanupTmp()
       4: cmd/xl-storage.go:251:cmd.newXLStorage()
       3: cmd/object-api-common.go:61:cmd.newStorageAPI()
       2: cmd/format-erasure.go:673:cmd.initStorageDisksWithErrors.func1()
       1: github.com/minio/pkg@v1.7.5/sync/errgroup/errgroup.go:123:errgroup.(*Group).Go.func1()

API: SYSTEM()
Time: 23:10:27 UTC 06/17/2024
Error: unable to create (/export/.minio.sys/tmp) file access denied, drive may be faulty please investigate (*fmt.wrapError)
       6: internal/logger/logger.go:258:logger.LogIf()
       5: cmd/prepare-storage.go:95:cmd.bgFormatErasureCleanupTmp()
       4: cmd/xl-storage.go:251:cmd.newXLStorage()
       3: cmd/object-api-common.go:61:cmd.newStorageAPI()
       2: cmd/format-erasure.go:673:cmd.initStorageDisksWithErrors.func1()
       1: github.com/minio/pkg@v1.7.5/sync/errgroup/errgroup.go:123:errgroup.(*Group).Go.func1()
ERROR Unable to use the drive /export: drive access denied: Invalid arguments specified
```

```console
Jun 17 16:09:44 nas1 systemd[1]: Created slice kubepods-burstable-podbd21dd4f_6fcb_4145_a4e4_cf56010280fe.slice - libcontainer container kubepods-burstable-podbd21dd4f_6fcb_4145_a4e4_cf56010280fe.slice.
Jun 17 16:09:44 nas1 kernel: kube-bridge: port 1(veth7dd6ad41) entered blocking state
Jun 17 16:09:44 nas1 kernel: kube-bridge: port 1(veth7dd6ad41) entered disabled state
Jun 17 16:09:44 nas1 kernel: veth7dd6ad41: entered allmulticast mode
Jun 17 16:09:44 nas1 kernel: veth7dd6ad41: entered promiscuous mode
Jun 17 16:09:44 nas1 kernel: kube-bridge: port 1(veth7dd6ad41) entered blocking state
Jun 17 16:09:44 nas1 kernel: kube-bridge: port 1(veth7dd6ad41) entered forwarding state
Jun 17 16:09:44 nas1 systemd[1]: Started cri-containerd-a024f763a8fbfa8a3ff7508069577eb34c349c1c40212f3abf857a94f9d95890.scope - libcontainer container a024f763a8fbfa8a3ff7508069577eb34c349c1c40212f3abf857a94f9d95890.
Jun 17 16:09:45 nas1 systemd[1]: Started cri-containerd-51bbdb14c073b23ce83c2287a830cbc999f2cb059b5d968ca369dab7d76a1187.scope - libcontainer container 51bbdb14c073b23ce83c2287a830cbc999f2cb059b5d968ca369dab7d76a1187.
Jun 17 16:09:45 nas1 systemd[1]: cri-containerd-51bbdb14c073b23ce83c2287a830cbc999f2cb059b5d968ca369dab7d76a1187.scope: Deactivated successfully.
Jun 17 16:09:46 nas1 systemd[1]: Started cri-containerd-ef23ecbacccf6853320827df1b9e3d4b988cbd2f3a87c05a67bba385845e5306.scope - libcontainer container ef23ecbacccf6853320827df1b9e3d4b988cbd2f3a87c05a67bba385845e5306.
Jun 17 16:09:46 nas1 systemd[1]: cri-containerd-ef23ecbacccf6853320827df1b9e3d4b988cbd2f3a87c05a67bba385845e5306.scope: Deactivated successfully.
Jun 17 16:09:46 nas1 systemd[1]: run-k3s-containerd-io.containerd.runtime.v2.task-k8s.io-ef23ecbacccf6853320827df1b9e3d4b988cbd2f3a87c05a67bba385845e5306-rootfs.mount: Deactivated successfully.
Jun 17 16:09:47 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:49 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:50 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:50 nas1 systemd[1]: Started cri-containerd-869fc104395058074b0a65fb308e9b4d9be8d7cdd66aed933cd7548646ecf917.scope - libcontainer container 869fc104395058074b0a65fb308e9b4d9be8d7cdd66aed933cd7548646ecf917.
Jun 17 16:09:50 nas1 systemd[1]: cri-containerd-869fc104395058074b0a65fb308e9b4d9be8d7cdd66aed933cd7548646ecf917.scope: Deactivated successfully.
Jun 17 16:09:50 nas1 systemd[1]: run-k3s-containerd-io.containerd.runtime.v2.task-k8s.io-869fc104395058074b0a65fb308e9b4d9be8d7cdd66aed933cd7548646ecf917-rootfs.mount: Deactivated successfully.
Jun 17 16:09:51 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:52 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:53 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:55 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:59 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:01 nas1 systemd[1]: Starting sysstat-collect.service - system activity accounting tool...
Jun 17 16:10:01 nas1 systemd[1]: sysstat-collect.service: Deactivated successfully.
Jun 17 16:10:01 nas1 systemd[1]: Finished sysstat-collect.service - system activity accounting tool.
Jun 17 16:10:05 nas1 systemd[1]: Started cri-containerd-0fddf9e0f0bbeeea02fada09ca9d7a7b74bc8f8bec20534b32de8b1c8e8b9dfd.scope - libcontainer container 0fddf9e0f0bbeeea02fada09ca9d7a7b74bc8f8bec20534b32de8b1c8e8b9dfd.
Jun 17 16:10:05 nas1 systemd[1]: cri-containerd-0fddf9e0f0bbeeea02fada09ca9d7a7b74bc8f8bec20534b32de8b1c8e8b9dfd.scope: Deactivated successfully.
Jun 17 16:10:05 nas1 systemd[1]: run-k3s-containerd-io.containerd.runtime.v2.task-k8s.io-0fddf9e0f0bbeeea02fada09ca9d7a7b74bc8f8bec20534b32de8b1c8e8b9dfd-rootfs.mount: Deactivated successfully.
Jun 17 16:10:07 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:17 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:18 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:19 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:19 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:20 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:20 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:21 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:21 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:22 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:22 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:23 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:24 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:24 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
```Jun 17 16:09:44 nas1 systemd[1]: Created slice kubepods-burstable-podbd21dd4f_6fcb_4145_a4e4_cf56010280fe.slice - libcontainer container kubepods-burstable-podbd21dd4f_6fcb_4145_a4e4_cf56010280fe.slice.
Jun 17 16:09:44 nas1 kernel: kube-bridge: port 1(veth7dd6ad41) entered blocking state
Jun 17 16:09:44 nas1 kernel: kube-bridge: port 1(veth7dd6ad41) entered disabled state
Jun 17 16:09:44 nas1 kernel: veth7dd6ad41: entered allmulticast mode
Jun 17 16:09:44 nas1 kernel: veth7dd6ad41: entered promiscuous mode
Jun 17 16:09:44 nas1 kernel: kube-bridge: port 1(veth7dd6ad41) entered blocking state
Jun 17 16:09:44 nas1 kernel: kube-bridge: port 1(veth7dd6ad41) entered forwarding state
Jun 17 16:09:44 nas1 systemd[1]: Started cri-containerd-a024f763a8fbfa8a3ff7508069577eb34c349c1c40212f3abf857a94f9d95890.scope - libcontainer container a024f763a8fbfa8a3ff7508069577eb34c349c1c40212f3abf857a94f9d95890.
Jun 17 16:09:45 nas1 systemd[1]: Started cri-containerd-51bbdb14c073b23ce83c2287a830cbc999f2cb059b5d968ca369dab7d76a1187.scope - libcontainer container 51bbdb14c073b23ce83c2287a830cbc999f2cb059b5d968ca369dab7d76a1187.
Jun 17 16:09:45 nas1 systemd[1]: cri-containerd-51bbdb14c073b23ce83c2287a830cbc999f2cb059b5d968ca369dab7d76a1187.scope: Deactivated successfully.
Jun 17 16:09:46 nas1 systemd[1]: Started cri-containerd-ef23ecbacccf6853320827df1b9e3d4b988cbd2f3a87c05a67bba385845e5306.scope - libcontainer container ef23ecbacccf6853320827df1b9e3d4b988cbd2f3a87c05a67bba385845e5306.
Jun 17 16:09:46 nas1 systemd[1]: cri-containerd-ef23ecbacccf6853320827df1b9e3d4b988cbd2f3a87c05a67bba385845e5306.scope: Deactivated successfully.
Jun 17 16:09:46 nas1 systemd[1]: run-k3s-containerd-io.containerd.runtime.v2.task-k8s.io-ef23ecbacccf6853320827df1b9e3d4b988cbd2f3a87c05a67bba385845e5306-rootfs.mount: Deactivated successfully.
Jun 17 16:09:47 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:49 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:50 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:50 nas1 systemd[1]: Started cri-containerd-869fc104395058074b0a65fb308e9b4d9be8d7cdd66aed933cd7548646ecf917.scope - libcontainer container 869fc104395058074b0a65fb308e9b4d9be8d7cdd66aed933cd7548646ecf917.
Jun 17 16:09:50 nas1 systemd[1]: cri-containerd-869fc104395058074b0a65fb308e9b4d9be8d7cdd66aed933cd7548646ecf917.scope: Deactivated successfully.
Jun 17 16:09:50 nas1 systemd[1]: run-k3s-containerd-io.containerd.runtime.v2.task-k8s.io-869fc104395058074b0a65fb308e9b4d9be8d7cdd66aed933cd7548646ecf917-rootfs.mount: Deactivated successfully.
Jun 17 16:09:51 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:52 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:53 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:55 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:09:59 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:01 nas1 systemd[1]: Starting sysstat-collect.service - system activity accounting tool...
Jun 17 16:10:01 nas1 systemd[1]: sysstat-collect.service: Deactivated successfully.
Jun 17 16:10:01 nas1 systemd[1]: Finished sysstat-collect.service - system activity accounting tool.
Jun 17 16:10:05 nas1 systemd[1]: Started cri-containerd-0fddf9e0f0bbeeea02fada09ca9d7a7b74bc8f8bec20534b32de8b1c8e8b9dfd.scope - libcontainer container 0fddf9e0f0bbeeea02fada09ca9d7a7b74bc8f8bec20534b32de8b1c8e8b9dfd.
Jun 17 16:10:05 nas1 systemd[1]: cri-containerd-0fddf9e0f0bbeeea02fada09ca9d7a7b74bc8f8bec20534b32de8b1c8e8b9dfd.scope: Deactivated successfully.
Jun 17 16:10:05 nas1 systemd[1]: run-k3s-containerd-io.containerd.runtime.v2.task-k8s.io-0fddf9e0f0bbeeea02fada09ca9d7a7b74bc8f8bec20534b32de8b1c8e8b9dfd-rootfs.mount: Deactivated successfully.
Jun 17 16:10:07 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:17 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:18 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:19 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:19 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:20 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:20 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:21 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:21 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:22 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:22 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:23 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:24 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
Jun 17 16:10:24 nas1 kernel: IPVS: rr: TCP 192.168.0.200:9000 - no destination available
```

**Comment by jadedeane** [comment]
Deleted MinIO, fresh install (inclusive of MinIO dataset for storage), same settings:
```console
root@nas1[~]# k3s kubectl -n ix-minio logs -f minio-797cf5946f-tnkb8
Defaulted container "minio" out of: minio, minio-init-01-permissions (init)

API: SYSTEM()
Time: 23:45:21 UTC 06/17/2024
Error: unable to rename (/export/.minio.sys/tmp -> /export/.minio.sys/tmp-old/d6c884c4-5768-4ed4-af2d-6cfd8bef8cee) file access denied, drive may be faulty please investigate (*fmt.wrapError)
       6: internal/logger/logger.go:258:logger.LogIf()
       5: cmd/prepare-storage.go:88:cmd.bgFormatErasureCleanupTmp()
       4: cmd/xl-storage.go:251:cmd.newXLStorage()
       3: cmd/object-api-common.go:61:cmd.newStorageAPI()
       2: cmd/format-erasure.go:673:cmd.initStorageDisksWithErrors.func1()
       1: github.com/minio/pkg@v1.7.5/sync/errgroup/errgroup.go:123:errgroup.(*Group).Go.func1()

API: SYSTEM()
Time: 23:45:21 UTC 06/17/2024
Error: unable to create (/export/.minio.sys/tmp) file access denied, drive may be faulty please investigate (*fmt.wrapError)
       6: internal/logger/logger.go:258:logger.LogIf()
       5: cmd/prepare-storage.go:95:cmd.bgFormatErasureCleanupTmp()
       4: cmd/xl-storage.go:251:cmd.newXLStorage()
       3: cmd/object-api-common.go:61:cmd.newStorageAPI()
       2: cmd/format-erasure.go:673:cmd.initStorageDisksWithErrors.func1()
       1: github.com/minio/pkg@v1.7.5/sync/errgroup/errgroup.go:123:errgroup.(*Group).Go.func1()
ERROR Unable to use the drive /export: drive access denied: Invalid arguments specified
```

I'm at a loss here.

**Comment by jadedeane** [comment]
Completely removed apps (i.e., unset pool, removed 'ix-applications' dataset', and selected pool again to re-create).

Installed MinIO again, and all is well.

**Comment by stavros-k** [comment]
Hmm, glad its working now.
I'm not really sure what happened there.
Please let me know if you see such issue again.

Thanks

**Comment by reitermarkus** [comment]
I'm having the same issue following the S3 to MinIO migration guide. I am currently still on Bluefin.

The login page takes ages to load in the first place. After it has loaded and I try logging in, I get the same timeout error. The domain is configured in the app wizard and the login page is using the correct certificate.

**Comment by reitermarkus** [comment]
@stavros-k, can this be reopened or should I open a new issue?
